configs/localization/ssn/README.md

```diff
@@ -24,6 +24,7 @@ According to the [Linear Scaling Rule](https://arxiv.org/abs/1706.02677), you ma
 e.g., lr=0.01 for 4 GPUs * 2 video/gpu and lr=0.08 for 16 GPUs * 4 video/gpu.
 2. Since SSN utilizes different structured temporal pyramid pooling methods at training and testing, please refer to [ssn_r50_450e_thumos14_rgb_train](/configs/localization/ssn/ssn_r50_450e_thumos14_rgb_train.py) at training and [ssn_r50_450e_thumos14_rgb_test](/configs/localization/ssn/ssn_r50_450e_thumos14_rgb_test.py) at testing.
 3. We evaluate the action detection performance of SSN, using action proposals of TAG. For more details on data preparation, you can refer to thumos14 TAG proposals in [Data Preparation](/docs/data_preparation.md).
+4. The reference performance is evaluated with `ResNet50` backbone, while the initial performance is evaluated with `BNInception` backbone. The comparison of MMAction2 SSN and the reference SSN are carried out under the same setting and data.
```

Comment:
`initial performance` is not exact

Reply:
4. The SSN in MMAction2 is evaluated with `ResNet50` backbone, while the reference SSN is evaluated with `BNInception` backbone. Other than backbone, the comparison of MMAction2 SSN and the reference SSN is carried out under the same setting and data.

Reply:
@tony2016uestc I updated the description. Hope this version is clearer

Reply:
OK, thank you very much!